src/app/EventPathParams.h

```diff
@@ -22,6 +22,11 @@
 
 namespace chip {
 namespace app {
+/**
+ *
+ * @brief This is used to record IM event path, and can pass event path info between IM and ember library.
```

Comment:
```suggestion
 * @brief This is used to record an M event path.
```
Again, why would one one "record" a path?